addons/hr_attendance/views/hr_attendance_view.xml

```diff
@@ -366,19 +366,23 @@
         <field name="tag">hr_attendance_greeting_message</field>
     </record>
 
-    <record id="hr_attendance_action_install_kiosk_pwa" model="ir.actions.client">
-        <field name="name">Attendance Kiosk</field>
-        <field name="target">new</field>
-        <field name="res_model">hr.attendance</field>
-        <field name="tag">install_kiosk_pwa</field>
-        <field name="context">{'app_id': 'hr_attendance', 'footer': false}</field>
+    <record model="ir.actions.server" id="open_kiosk_url">
+        <field name="name">Open Kiosk Url</field>
+        <field name="model_id" ref="hr_attendance.model_res_company"/>
+        <field name="path">attendance-kiosk</field>
+        <field name="binding_model_id" ref="hr_attendance.model_res_company"/>
+        <field name="state">code</field>
+        <field name="code">
+            action = model._action_open_kiosk_mode()
+        </field>
+        <field name="groups_id" eval="[(4, ref('hr_attendance.group_hr_attendance_manager'))]"/>
```

Comment:
Not sure but i think we need an upgrade script for this 
can you please have a look?

Reply:
Helloo @thsh-odoo 

Its working fine. I tested it by upgrading a db in local.
Please let me know if I have missed something.

Thanks :blush:

Reply:
Keep this conversation open for the reviewers regarding upgrade